### ISSUE TITLE
Narrator is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed : A11y_.NET Core_RecipesAppMyRecipesNew_Search box_screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#22845]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags
#A11yTCS;#A11yMAS;#WCAG1.3.1;#Devtool;#E2E_.NETCore_Win32_Jun2024;#.NET Core;#Name Role Value;#DesktopApp;#Win32;#FTP;#A11ySev2;#AILimited;#STP;#ScreenReader;#Narrator;#NVDA;#Element:Editfield;

## Environment Details:
Application: .NET Core
Version .NET 9
Operating System: Window 11 version 23H2 OS build 22631.2861
NVDA version 2024.1
Narrator Version win11

## Repro Steps:

1. Turn on Narrator screen reader
2. Install and open ".NET MAUI" app.
3. TAB till "Search recipe" edit filed search some thing 
6. Observe narrator is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed.

## Actual Result:
Narrator is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed
## Expected Result:
Narrator should announce the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed.
## User Impact:
Screen reader user will not known for their search result is found or not.
### Attachment
Nattator
![22845_Narrator](https://github.com/dotnet/maui/assets/124254981/9a6a683a-8ca5-42fe-a7e4-ff5970ae9244)
NVDA
![22845_NVDA](https://github.com/dotnet/maui/assets/124254981/52d54f11-ac2c-49ea-86ca-6393aaf0257b)
Code Snippet
![22845_Code snippet](https://github.com/dotnet/maui/assets/124254981/d37837aa-6801-4b42-80f2-15afc5a1b8a3)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio 17.11.0 Preview 2.0 (8.0.40&8.0.21). Can repro on Windows platform with sample project.

**Comment by kapilvaishna** [comment]
Issue is not repro with for both screen reader in version 3(3) so closing the bug.

https://github.com/user-attachments/assets/63d6f70c-eec5-4577-a8cd-d5629e7035af


https://github.com/user-attachments/assets/1e4b5b54-12bc-4e98-b8c2-bcb2e3ac3d2b
